SPECS/http-nbd-transfer.spec

```
@@ -33,5 +33,11 @@ make
 %{_libdir}/nbdkit/plugins/nbdkit-multi-http-plugin.so
 
 %changelog
+* Mon Jan 9 2023 Ronan Abhamon <ronan.abhamon@vates.fr> - 1.1.0-1
+- HTTP server can reuse binding address now
+- Notify when HTTP server is ready
+- Better error handling and command logs
+- SIGTERM can be safely sent to NBD server
+
 * Tue May 3 2022 Ronan Abhamon <ronan.abhamon@vates.fr> - 1.0.0-1
```

Didn't notice this one in the past:

```suggestion
* Tue May 03 2022 Ronan Abhamon <ronan.abhamon@vates.fr> - 1.0.0-1
```